aten/src/ATen/native/quantized/cpu/qconcat.cpp

```diff
@@ -122,5 +124,19 @@ static auto registry =
                 TensorTypeId::QuantizedCPUTensorId));
 
 } // namespace
-} // namespace native
-} // namespace at
+
+Tensor quantized_cat(TensorList qxs, int64_t dim) {
+  double _scale = qxs[0].q_scale();
+  int64_t _zero_point = qxs[0].q_zero_point();
```

Comment:
check for qscheme?
and check that scale and zero_point match for all inputs?

Reply:
I don't think we need to check as we have a dequant/quant steps. Should I still have the check?

Reply:
`q_scale` is only valid for `PerTensorAffine` quantized tensor

Reply:
we have check in `quantized_cat_impl` but I think it's probably too late if you are calling `q_scale` here